src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -153,22 +162,30 @@ def acr_show(registry_name, resource_group_name=None):
     return client.get(resource_group_name, registry_name)
 
 
-def acr_update_get(client):  # pylint: disable=unused-argument
+def acr_update_get(client,  # pylint: disable=unused-argument
+                   registry_name,
+                   resource_group_name=None):
     """Returns an empty RegistryUpdateParameters object.
+    :param str registry_name: The name of container registry
+    :param str resource_group_name: The name of resource group
     """
-    return RegistryUpdateParameters()
+    try:
+        registry_sku_validation(registry_name, resource_group_name)
+        return ManagedRegistryUpdateParameters()
+    except:  # pylint: disable=bare-except
+        return BasicRegistryUpdateParameters()
 
 
 def acr_update_custom(instance,
+                      sku=None,
                       storage_account_name=None,
                       admin_enabled=None,
                       tags=None):
+    if sku is not None:
```

Comment:
`sku` has an options list so it doesn't matter, but it couldn't be empty either. For update path there are additional validation methods for empty check, etc.